EMML/1-1000/EMML50.xml

```diff
@@ -231,7 +231,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                      </msItem>
 
                     <msItem xml:id="ms_i2.19">
-                       <locus from="150"></locus>
+                       <locus target="#150"></locus>
```

Comment:
target is to one specific page only, so it must be 150r if you are sure, or if it is 150r to 150v then you need from-to